### PR TITLE
Various tweaks and additions

### LINKED ISSUE #2620: Put ESCAPE pods on the ODP
#### Description of issue
When the Covenant go through the umbilical, or, if they block it off, the marines, unless they get to the Pelican (which can easily be destroyed) they are, essentially, stranded. It doesn't make sense to not include at least two.

#### Difference between expected and actual behavior
Urist McMarine [squadcomm] says, "Get to the escape pods!"
Urist McCommandingOfficer [squadcomm] says, "It wasn't in the budget! We'll have to go down with the ODP!"

#### Steps to reproduce 
Covenant Activate the Nuke/Slipspace Core Meltdown
Look for escape pods
There aren't any.
Cry.
Die.

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of the issue being looked at sooner.

- [X] Issue could be reproduced at least once
- [X] Issue could be reproduced by different players
- [X] Issue could be reproduced in multiple rounds
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] [Couldn't find an existing issue about this](https://github.com/HaloSpaceStation/HaloSpaceStation13/issues)

### PR DESCRIPTION
**Pull request by X0-11**
:cl: XO-11
tweak: species speed curve standardised
tweak: The scanners in the Invasion gamemode are now more consistent in taking explosion damage.
tweak: Vehicles no longer take multiple instances of explosion damage.
tweak: Vehicles have now been tweaked to be less resistant to small arms fire in cases where it makes sense.
tweak: tweaks movement sound effects to not spam them every step.
tweak: Makes barbed wire not impact fliers.
tweak: Escape pods are now present on the ODP
/:cl:
closes #2620
closese #2623

### CODE REVIEW

**Comment by caelaislinn** [comment]
Got some missing icon errors

File: code/modules/halo/weapons/covenant/brute.dm [hunk 1]
```diff
@@ -390,7 +391,7 @@
 	while(grensleft > 0)
 
 		//create an image of a grenade and tweak it a bit
-		var/image/gren = image('code/modules/halo/weapons/icons/jiralhanae_obj.dmi', "bruteshot_gren")
+		var/image/gren = image('code/modules/halo/covenant/species/jiralhanae/jiralhanae_obj.dmi', "bruteshot_gren")
```
Comment (caelaislinn) [inline code/modules/halo/weapons/covenant/brute.dm:394]: Do these changes fix #2419 and #2615 ?

**Review by caelaislinn — APPROVED** [review]
I'd rather new features like sparrowhawk and banshee code go into a separate PR and branch but it's good to merge

**Comment by caelaislinn** [comment]
2623 didn't autoclose because you mispelt "close" 🤣

File: code/modules/halo/structures/_destructible.dm [hunk 2]
```diff
@@ -143,10 +143,6 @@
 		if(mover.throwing)
 			return climbable
 
-		//we are something small enough to slip around obstacles, but not a bullet
-		if(mover.checkpass(PASSTABLE))
-			return 1
-
```
Comment (caelaislinn) [inline code/modules/halo/structures/_destructible.dm:149]: What was this removed for?

Comment (X0-11) [reply code/modules/halo/structures/_destructible.dm:149]: Dodge rolling was bypassing defenses too easily but I still wanted to allow rolling to move over tables.

**Pull request merged**